@google-cloud/storage/bucket/index.d.ts

```diff
@@ -0,0 +1,102 @@
+/// <reference types="node" />
+
+import { ReadStream } from "fs";
+import { Channel, ChannelConfig } from "../channel";
+import { File } from "../file";
+import {
+  Acl,
+  ApiResponse,
+  UploadOptions
+} from "../storage";
+
+declare module "@google-cloud/storage/bucket" {
```

Comment:
My comment about "Require" is not correct. Although you can use ambient module declaration with es6 style import, It will be better to pull these off to top levels.